inflate_fast_amd64_linkaudit_test.go

```
@@ -0,0 +1,648 @@
+	}
+	if p, err := lookPath("go"); err == nil {
+		return p, true
+	}
+	return "", false
+}
+
+// amd64LinkAuditGoTool locates the `go` binary needed for `go tool objdump`:
+// the GOROOT environment variable, then runtime.GOROOT()/bin/go, then PATH.
+func amd64LinkAuditGoTool() (string, bool) {
+	return amd64LinkAuditGoToolFrom(
+		[]string{os.Getenv("GOROOT"), runtime.GOROOT()}, exec.LookPath)
```

golangci-lint (2.12.2)

[error] SA1019: runtime.GOROOT has been deprecated since Go 1.24: The root used during the Go build will not be meaningful if the binary is copied to another machine. Use the system path to locate the “go” binary, and use “go env GOROOT” to find its GOROOT. 

(staticcheck)